src/net/sourceforge/kolmafia/request/CampgroundRequest.java

```diff
@@ -554,11 +548,29 @@ public static AdventureResult getCrop() {
 
   public static List<AdventureResult> getCrops() {
     var list = new ArrayList<AdventureResult>();
```

Comment:
I want the resulting list to be in the order crops appear in CROPS - which is an (ordered) array.
For gardens with only a single crop, it is moot, but for the rock garden, I want the first crop to be plot1, the second crop to be plot2, and the third crop to be plot3

Reply:
So LinkedHashSet then

Reply:
I'm not going to change the List into a Set for the reasons I mentioned.
Since it was previously approved and all I added was additional tests, I'm going to merge this.